Library/Formula/swagger-codegen.rb

```diff
@@ -0,0 +1,33 @@
+class SwaggerCodegen < Formula
+  desc "Generation of client and server from Swagger definition"
+  homepage "http://swagger.io/swagger-codegen/"
+  url "https://github.com/swagger-api/swagger-codegen/archive/v2.1.4.tar.gz"
+  sha256 "b4965158fb8abbebdfcaab2fdc0c5556001c587a526279d44ea9878f7ef9ef4c"
+  head "https://github.com/swagger-api/swagger-codegen.git"
+
+  depends_on :java => "1.7"
```

Comment:
According to https://github.com/swagger-api/swagger-codegen#prerequisites it only works with 1.7.

My first attempt was with `"1.7+"` and it actually made the testbot fail.